aplus/settings.py

```diff
@@ -249,9 +249,10 @@
 ]
 TIME_ZONE = 'EET'
 USE_I18N = True
-USE_L10N = True
+USE_L10N = False
 USE_TZ = True
-FORMAT_MODULE_PATH = 'aplus'
+DATETIME_FORMAT = 'd.m.Y l H:i'
+SHORT_DATE_FORMAT = 'd.m.Y'
```

Comment:
You could add a short comment explaining why we want to set `USE_L10N = False` by default.

When `DATETIME_FORMAT` is set, I suppose the related settings `TIME_FORMAT, DATE_FORMAT, SHORT_DATETIME_FORMAT` should also be set.

Reply:
Sure, that sounds reasonable.